### PR TITLE
Update tutorials files for FDK lesson3 and lesson4

### LINKED ISSUE #83: FDK tutorial lesson 4
One of the last changes of lesson 3 is to move a node at the knee. This change is missing in the finished  model that can be downloaded in Lesson 4. 

Thanks to @toerholm for finding this.

### PR DESCRIPTION
**Pull request by melund**
This fixes a lot the intermediate files in the tutorial. 

fixes: #83

### CODE REVIEW

File: ForceDependentKinematics/lesson4.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ realistic geometries of surfaces such as the femoral condyles, and we
 shall explore those in this lesson. We start from the model developed in
 the second lesson. If you did not manage to obtain a working model from
 the second lesson, then please download a new one
-{download}`here <Downloads/DemoSimpleKnee2.any>`.
+{download}`here <Downloads/DemoSimpleKnee3.any>`.
```
Comment (bkje14) [inline ForceDependentKinematics/lesson4.md:13]: Here it says that can download the model from the second lession, but the file is labeled as 3? is that intended or should it just say to download the one from lession 3?

File: ForceDependentKinematics/Downloads/DemoSimpleKnee4.any [hunk 2]
```diff
@@ -33,15 +34,26 @@ Main = {
               RGB = {0, 0, 1};
             };
           };
+        };  
+        
+        // Define a STL file as femoral condyle for the joint geometry. The tibial
```
Comment (bkje14) [inline ForceDependentKinematics/Downloads/DemoSimpleKnee4.any:39]: Downloading `DemoSimpleKnee4.any` does not include the femoral stl file - could we bundle those two perhaps?

**Review by bkje14 — CHANGES_REQUESTED** [review]
Build and checked the lession.
Downloaded demo 2,3 and 4 and tried them in ams.

demo 4 cant load due to a missing stl.

File: ForceDependentKinematics/Downloads/DemoSimpleKnee4.any (hunk 2 above)
Comment (melund) [reply ForceDependentKinematics/Downloads/DemoSimpleKnee4.any:39]: I will add that in a separate PR.

**Review by abrg-data-access — COMMENTED** [review]
Looks good.

**Pull request merged**